locales/nb-no.json

```diff
@@ -0,0 +1,797 @@
+{
+  "lang": "nb-no",
```

Comment:
@olemarius Are you sure this is correct? Looking at the language tag registry, it looks to me like Norwegian Bokmål should just be "nb".

Reply:
@WilcoFiers actually there's not really any naming conventions for locale filenames specifically, but it should at least include  both language and country codes. I noticed now that you use language_COUNTRY.json for brazil where they speak Portugese:

[pt_BR.json](https://github.com/dequelabs/axe-core/blob/develop/locales/pt_BR.json)

so in order to keep the filename naming convention consistent I think `nb_NO.json` would be better.

Reply:
Alright, works for me. Just to give a heads up, I'm still looking for a reviewer.

Reply:
@WilcoFiers  - nb_NO totally makes sense. Norway has multiple "official" languages (Bokmål (nb) and Nynorsk (nn) most common) so it makes sense to support nb_NO and nn_NO to support both.

(src: https://www.loc.gov/standards/iso639-2/php/langcodes-keyword.php?SearchTerm=Norwegian&SearchType=ALL=

Reply:
I see that there are some minor errors, possibly due to original danish texts, that need to be fixed;

- https://naob.no/ordbok/attributt
- beskeden
- delvist
- etc.

Will take some time to go through all of them asap and will make my fix suggestions here.

Reply:
Great, I'll go trough and update when you're done and rename to correct filename 👌🏼